docs/assets/styles.css

```diff
@@ -386,6 +385,20 @@ section.cover .cover-main {
     max-width: 100px;
     max-height: 100px;
   }
+
+  body .sidebar-toggle { background-color: #fff; }
+  body.close .sidebar-toggle {
+    background-color: transparent;
```

Comment:
maybe you can move this declaration directly under `body .sidebar-toggle {}`? (currently has background transparent and you are overriding it.

Reply:
Same, it's because I'm overriding a 3rd party CSS file